spec/models/vocab_sheet_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe 'VocabSheet', type: :model do
+  describe '#clear_vocab_sheet' do
+    it 'only deletes a vocab sheet older than 31 days' do
+      FactoryBot.create_list(:vocab_sheet, 5)
+      vocab_sheet = FactoryBot.create(:vocab_sheet, created_at: 1.year.ago, updated_at: 1.year.ago)
+      FactoryBot.create_list(:item, 3, vocab_sheet_id: vocab_sheet.id)
+
```

Comment:
It seems you are testing 2 things here, so it would be great to separate them into 2 tests.

1. `clear_vocab_sheet` deletes the sheet that is more than 31 days old and not the others - checking both the number of records and that the remaining ones have `updated_at` less than 31 days ago.

2. That when the vocab sheet is deleted, the items are too.

This is fine, it is just not super obvious what you are checkin and why based on the `it` block statement alone.

Reply:
i love that there are tests now.... so we can be more confident future devs won't make a mistake and delete all data.